src/graphql/context.ts

```diff
@@ -8,13 +9,35 @@ export interface IContext {
 }
 
 export class Context implements IContext {
-	private accountId: string;
+	private signedMessage: string;
 
-	constructor(accountId: string) {
-		this.accountId = accountId;
+	/**This constructor will recieve singed messages for any transactions
+	 * and return the address which the message came from to be used as
+	 * context variable in other scripts.
+	 */
+	constructor(signedMessage: string) {
+		this.signedMessage = signedMessage;
 	}
 
 	public getEtheriumAccountId(): string {
-		return this.accountId;
+		return this.findAccount();
+	}
+
+	private findAccount() {
+		const web3 = new Web3(
+			new Web3.providers.HttpProvider('http://localhost:7545')

```

Comment:
Look at UserDescriptorsService.ts. On the top of that file there is a call to a library called dotenv. This very call to make a new instance then gets a global envrionment variable to the blockchain. This lets us change it in one place and have it reflect all files uniformly (the address is stored in .env file in the root). Lets use the same technique for here.